onnxruntime/core/providers/webgpu/webgpu_context.cc

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include "webgpu_context.h"
```

Comment:
i think this is no need. Line 28 of this file should already containding this header file.